frontend/src/pages/Profile.tsx

```diff
@@ -202,10 +202,13 @@ const Profile = () => {
 
 
     return (
-        <div className="profile-page">
+        <div className="profile-page" style={{
+            backgroundColor: "#646463",
+            height: "100vh"
```

Comment:
Why have the height css?

Reply:
Without it there is a white rectangle at the bottom that ruins the visuals

Reply:
Understood